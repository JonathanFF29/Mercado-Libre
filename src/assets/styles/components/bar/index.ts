import styled, {css} from 'styled-components';

export const Bar = styled.div`
 ${({theme}) => css `
    background-color: ${theme.colors.yellowml};
    width:100%;
    text-align:center;
    height: 50px;
 `}
`