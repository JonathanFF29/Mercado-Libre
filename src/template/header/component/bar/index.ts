import styled, {css} from 'styled-components';

export const Bar = styled.div`
 ${({theme}) => css `
    display:flex;
    background-color: ${theme.colors.yellowml};
    justify-content:center;
    align-items: center;
    width:100%;
    height: 50px;
 `}
`