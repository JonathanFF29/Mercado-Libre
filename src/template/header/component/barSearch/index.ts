import styled, { css } from 'styled-components';

export const BarSearch = styled.div`
 ${({ theme }) => css`
    display:flex;
    background-color: ${theme.colors.whiteml};
    width: 60%;
    height:30px;
    input {
        border:none;
        font-size: 18px;
        padding-left: 10px;
        width:90%;
    },
    input:focus {
        border:none;
        outline: -webkit-focus-ring-color auto 0px;
        outline-width: none;
}
    },
 `}
`