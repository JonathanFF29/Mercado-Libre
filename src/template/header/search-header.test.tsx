import {configure, shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import React from 'react';
import { Bar } from './component/bar';

configure({adapter: new Adapter()})

describe('<SearchHeader>', () => {
    it('should should render ' , () => {
        const wrapper = shallow(<Bar/>)
        expect(wrapper).toMatchSnapshot();
    })
})