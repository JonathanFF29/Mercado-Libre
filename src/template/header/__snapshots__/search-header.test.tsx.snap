// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchHeader> should should render  1`] = `ShallowWrapper {}`;
